lib/clients/helm.js

```diff
@@ -385,7 +388,12 @@ class Helm {
   }
 
   async _installPrometheusOperator() {
-    return this._installGenericChart('prometheus-operator');
+    if(this.opsgenieToken === ''){
```

Comment:
Here you are still controlling whether or not enable opsgenie with an env var, please use here an `opsgenie` configuration option and include it explicitly in the example files.